.github/workflows/binaries.yml

```diff
@@ -12,6 +12,8 @@ jobs:
         os:
           - name: macos-latest
             binary: csvsql
+          - name: linux-latest
```

Comment:
I think you want ubuntu-latest here.

Reply:
Thanks; I added another commit. Please let me know if you want me to squash them, or if you will do this anyway when (hopelully) merging the PR.